tests/topotests/multicast_pim6_dual_stack/test_multicast_pim6_dual_stack.py

```diff
@@ -0,0 +1,1952 @@
+                            "/"
+                        )[0],
+                        "group_addr_range": GROUP_RANGE_1,
+                    }
+                ]
+            },
+        }
+    }
+
+    result = create_pim_config(tgen, topo, input_dict)
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step("send mld join (ffaa::1-5) from R1")
+    result = app_helper.run_join("i1", MLD_JOIN_RANGE_1, "r1")
+    assert result is True, "Testcase {}: Failed Error: {}".format(tc_name, result)
+
+    step("Send IGMP joins from DUT for group range 225.1.1.1-5")
+    input_join = {
+        "i1": topo["routers"]["i1"]["links"]["r1"]["interface"],
+    }
+
+    for recvr, recvr_intf in input_join.items():
+        result = app_helper.run_join(recvr, IGMP_JOIN_RANGE_1, join_intf=recvr_intf)
+        assert result is True, "Testcase {}: Failed Error: {}".format(tc_name, result)
+
+    step(
+        "IPv4 mld and IPv6 MLD join received on R1 from same interface"
+        'verify using "show ipv6 mldgroups"  and "show ipv6 mld groups"'
+    )
+
+    intf_r1_i1 = topo["routers"]["r1"]["links"]["i1"]["interface"]
+    result = verify_mld_groups(tgen, "r1", intf_r1_i1, MLD_JOIN_RANGE_1)
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    result = verify_igmp_groups(tgen, "r1", intf_r1_i1, IGMP_JOIN_RANGE_1)
+    assert result is True, "Testcase {}:Failed \n Error: {}".format(tc_name, result)
+
+    oif = topo["routers"]["r1"]["links"]["r2"]["interface"]
+    step(
+        "IPv4 and IPv6 (*,G) mroute and upstream created OIL"
+        'and IIF updated using "show ipv6 mroute json" "show ipv6 mroute json"'
+        '"show ipv6 pim upstream json" "show ipv6 upstream json"'
+    )
+
+    iif1 = topo["routers"]["r1"]["links"]["i1"]["interface"]
+    iif2 = topo["routers"]["r1"]["links"]["i1"]["interface"]
+    for iif, grp_addr in zip([iif2, iif1], [IGMP_JOIN_RANGE_1, MLD_JOIN_RANGE_1]):
+        step("r1: Verify upstream IIF interface")
+        result = verify_upstream_iif(tgen, "r1", oif, "*", grp_addr)
+        assert result is True, ASSERT_MSG.format(tc_name, result)
+
+        step("r1: Verify ip mroutes")
+        result = verify_mroutes(tgen, "r1", "*", grp_addr, oif, iif)
+        assert result is True, ASSERT_MSG.format(tc_name, result)
+
+    step(
+        "Send IPv4 and IPv6 multicast traffic from FRR4 same interface"
+        "group range (FF05::1 -FF05::5 and 226.1.1.1 -226.1.1.5"
+    )
+
+    result = app_helper.run_traffic("i7", MLD_JOIN_RANGE_1, "r4")
+    assert result is True, "Testcase {}: Failed Error: {}".format(tc_name, result)
+
+    result = app_helper.run_traffic("i8", IGMP_JOIN_RANGE_1, "r4")
+    assert result is True, "Testcase {} : Failed Error: {}".format(tc_name, result)
+
+    step(
+        "(S,G) IPv4 and IPv6 mroute /upsteam created on FRR1 and FRR4"
+        '"IIF and OIL updated using show ipv6 mroute json" "show ipv6 mroute json"'
```

Comment:
Is this repeated occurrences of "show ipv6 mroute" in the sentence intentional?

Reply:
It is typo, updated with "show ip mroute json" and "show ipv6 mroute json", we are verifying both. Thanks